x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -159,40 +143,6 @@ func (je *journeyEnricher) enrichSynthEvent(event *beat.Event, se *SynthEvent) e
 	return jobErr
 }
 
-func (je *journeyEnricher) createSummary(event *beat.Event) error {
-	// In case of syntax errors or incorrect runner options, the Synthetics
-	// runner would exit immediately with exitCode 1 and we do not set the duration
-	// to inform the journey never ran
-	if !je.start.IsZero() {
-		duration := je.end.Sub(je.start)
-		eventext.MergeEventFields(event, mapstr.M{
-			"monitor": mapstr.M{
-				"duration": mapstr.M{
-					"us": duration.Microseconds(),
-				},
-			},
-		})
-	}
-	eventext.MergeEventFields(event, mapstr.M{
-		"url": je.urlFields,
-		"event": mapstr.M{
-			"type": "heartbeat/summary",
-		},
-		"synthetics": mapstr.M{
-			"type":    "heartbeat/summary",
-			"journey": je.journey,
-		},
-	})
-
-	// Add step count meta for log wrapper
-	eventext.SetMeta(event, logger.META_STEP_COUNT, je.stepCount)
```

Comment:
Removing this has effect on the Logger where the steps would never be logged. we need this logic to be on summarizer.

Reply:
++ bringing back to plugerr.go where we count steps